File: 1 Revalida en python/5 Funciones/4.py
 #funcion para clasificar la clase de barcos
def clase(c):
	if(c=='D'):
		print ("Destructor")
	elif(c=='F'or c=='f'):
		print("Fragata")
	elif(c=='M'or c=='m'):
		print("Misilera")
	elif(c=='B'or c=='b'):
		print("Buque")
	else:
		print ("No exixte es clase de barco")
c=raw_input("ingrese un caracter:")
print clase(c)

File: 2 Revalida en C++/5 Funciones/4.cpp
///funcion para clasificar la clase de barcos
#include <bits/stdc++.h>
using namespace std;
void clase(char c)
{
    if(c=='D'||c=='d')
		cout<<"Destructor";
    else if(c=='F'||c=='f')
		cout<<"Fragata";
    else if(c=='M'||c=='m')
		cout<<"Misilera";
    else if(c=='B'||c=='b')
		cout<<"Buque";
	else
		cout<<"No exixte esa clase de barco";
}
int main()
{
    char c;
    cout<<"ingrese un caracteren :";
    cin>>c;
    clase(c);
    return 0;
}
